### ISSUE TITLE
[ALL PAGES] Add alt text for all images and graphics on all pages (improve SEO)

### ISSUE THREAD

**Issue by initialdpw** [hmcc-global/hmccaa-web#441]
Please see below for all the alt text descriptions for all images and graphics on hmcc page. Let me know if I'm missing anything or you have any questions!

**MAIN PAGES:**

**_HOME:_**
- **Main first image:** people of all nations worshipping inside HMCC sanctuary
- **WELCOME image:** Wooden cross surrounded by a crowd of students in a circle inside the sanctuary
- **GET CONNECTED images:** 
	**- Building Blocks:** classroom full of young kids and teachers eating and having fun, representing our Building Blocks ministry
	**- Latitude/Velocity:** group of middle school and high school kids in festive attire, smiling and posing in fun ways, representing our Latitude and Velocity life stage
	**- Global Access:** international students celebrating someone's birthday and having fun, representing our Global Access ministry
	**- Access:** group of undergraduate students sitting on a staircase smiling and posing in unison, representing our Access ministry
	**- Impact:** group of graduate students at an outdoor sports event with group t-shirts and large flag, representing our Impact ministry
	**- Focus:** group of adults in church lobby celebrating a baptism of a fellow member, representing our Focus ministry
	**- Covenant:** a long table of married couples eating dinner together at someone's home, representing our Covenant ministry
- **GET INVOLVED graphics:** 
	**- CONNECT:** graphic of circular dots showing a gray dot connected to other blue dots
	**- JOIN LG:** graphic of blue circular dots arranged in a circle with one dot being gray
	**- MEMBER:** graphic of various blue dots representing unity
**- UPCOMING EVENTS image:** fun bright outdoor sports event with a girl and guy greeting, smiling, and inviting you to join

_**ABOUT US:**_
**- OUR STORY image:** top view of our sanctuary looking down at the audience and the stage with the pastor preaching in the middle
**- OUR LEADERSHIP images:** see OUR TEAM side page
**- OUR STRATEGY graphics:** graphic of several blue arrows in different size and directions representing Gather, Grow, and Go
**- FULL BANNER image:** large full-width banner of over 60 married couples and kids outdoor smiling at the camera
**- OUR PARTNERS image:** undergraduate students on campus witnessing and telling others about church and faith at a club fair

_**NEXT STEPS:**_
**- LIFE GROUP image:** vibrant close-knit group of undergraduate students holding cute stuffed animals of various sizes and smiling beautifully
**- T-CLASSES image:** a man teaching a class of students in a room with a microphone and PowerPoint slides
**- MINISTRY TEAM icons:** see MINISTRY TEAMS side page
**- FULL BANNER image:** large full-width banner of roughly 100 undergraduate students wearing their life group t-shirts and posing in their own life group styles
**- MISSIONS image:** a few of our members in red t-shirts on a missions trip to Peru, teaching and playing with kids at the table

_**WATCH ONLINE:**_
**- LIVE STREAM image:** a laptop on a bright wooden chest drawer live-streaming a sermon from HMCC

_**GIVING:**_
**- WHY WE GIVE image:** a woman coming before a large wooden cross to stick a sticker on it, as a symbolism of laying her sins on the cross
**- WAYS TO GIVE icons:** 
	- minimalistic icon of a laptop 
	- minimalistic icon of an envelop or mail

_**I'M NEW (new version):**_
**- WELCOME image:** church congregation meeting new people and talking to each other in the sanctuary after celebration
**- CHURCH MAP:** map to the church and parking space locations, with the church located between East Huron Street and East Ann Street
**- COMMON QUESTIONS image:** rectangular white building with a blue sign in front saying Harvest Mission Community Church
**- LIFE STAGES images:**
	**- families:** a few married couples and their kids at an outdoor hangout event
	**- single adults:** a diverse group of single adults at someone's home around a couch
	**- grad students:** a group of graduate students at someone's home smiling at the camera
	**- undergrads:** a few undergraduate student friends standing next to each other with matching color clothing
	**- international students:** a group of Asian international students standing together outside near a lake
**- FULL BANNER image:** large full-width banner of the entire church at an outdoor sports event with their respective life group shirts and large flags

//

**SIDE PAGES**

**_OUR TEAM:_**
**- MEET THE ELDERS images:** profile picture of [insert name]
**- MEET THE DEACONS images:** profile picture of [insert name]
_[note: unsure if we want to say their ethnicity and gender, i.e. profile picture of Pastor Pete Dahlem, a white man / profile picture of Pastor Joshua Yang, a Korean man]?_

**_MEMBERSHIP:_**
**- MAIN image:** a group of new members on stage being recognized by the whole church

**_BAPTISM:_**
**- MAIN image:** pastor baptizing a new believer in a water tank along with close friends nearby

**_LIFE GROUP:_**
**- MAIN video:** video explaining what Life Group is all about
**- SMALLER image 1/4:** group of adults and kids outdoor at a sports competition having fun
**- SMALLER image 2/4:** group of students at a retreat cafeteria eating together in large circular tables
**- SMALLER image 3/4:** a life group huddling in a circle and praying for one another
**- SMALLER image 4/4:** a group of students on a staircase posing in unison and smiling beautifully
**- Circular E icon:** graphic of the letter E, representing one of the five values of biblical community

**_MINISTRY TEAMS:_**
**- MAIN image:** behind the scenes shot of the sound team and video ministry teams serving on a Sunday morning
**- SMALLER image 1/4:** a couple members of the art and design ministry team painting on large boards to be displayed in the church lobby
**- SMALLER image 2/4:** small kids doing energetic morning exercises along with their teachers at the church basement
**- SMALLER image 3/4:** various members of our host team welcoming and talking to new and returning people at the church lobby
**- SMALLER image 4/4:** servants team cutting and blowing grass away from the sidewalks
**- MINISTRY TEAM icons:**
	**- ADS:** black paint splatter on pale yellow square, representing our Art and Design Servants ministry team
	**- DOCS:** black camera on pale red square, representing our photo and video ministry team
	**- VM:** black video camera on pale purple square, representing our video recording ministry team
	**- BB:** black children blocks on pale orange square, representing our Building Blocks or children's ministry
	**- HOSTS:** black coffee cup with steam on pale light blue square, representing our Hospitality or Welcome team
	**- SERVANTS:** black broom on pale green square, representing our servants team
	**- WORSHIP:** black fire on pale blue square, representing our worship team

**_TRANSFORMATION CLASSES:_**
**- MAIN image:** someone walking up a long flight of steps with a book in hand
**- SMALLER image 1/4:** small groups of people around tables learning about discipleship
**- SMALLER image 2/4:** large class of people listening to a course about apologetics
**- SMALLER image 3/4:** small group of brother praying for each other during Freedom Class
**- SMALLER image 4/4:** small group of people discussing with each other on the Great Commission

**_HMCC PARTNERS:_**
**- PAST PROJECTS flag images:** flag of [country name]
**- SUPPORT WORKERS images:** picture of [ministry partner names] (and their family)

**_NEXT PAGE > MISSIONS SUPPORT:_**
**- MAIN image:** wooden world map with colorful pins at different locations

**_404 ERROR:_**
**- MAIN image:** top view of a spilled coffee drink splattered near a pair of white shoes
